services/meta/association.md

```diff
@@ -1,45 +1,75 @@
+in   | cmd.association.add        | object     |                | Add members to a specific association group. See [`association_members`](#definitions) for value definition.                                                                                        |
+in   | cmd.association.delete     | object     |                | Remove a single memeber from an association group. See [`association_members`](#definitions) for value definition.                                                                       |
+in   | cmd.association.delete_all | int        |                | Remove all memebers from an association group.                                                                                                                                           |
+in   | cmd.association.get_report | int        |                | Request report of a specific association group.                                                                                                                                          |
+out  | evt.association.report     | object     | `max_supports` | Reports members of a specific association. Property `max_supports` defines how many devices can be added to this group. See [`association_members`](#definitions) for value definition.  |
+in   | cmd.groups_info.get_report | null       |                | Z-Wave only. Request detailed information about supported association groups.                                                                                                            |
+out  | evt.groups_info.report     | object     |                | Z-Wave only. Returns a list of supported associations groups with descriptions. See [`association_info_object`](#definitions).                                                           |
+
+## Service Props
+
+Name             | Supported Values                                | Description                                                             |
+-----------------|-------------------------------------------------|-------------------------------------------------------------------------|
+`in_services`    | `["out_bin_switch", "out_lvl_switch", "color"]` | Zigbee only. List of services that can be controlled, e.g. on a lamp.   |
+`out_services`   | `["out_bin_switch", "out_lvl_switch", "color"]` | Zigbee only. List of services the device can control, e.g. on a button. |
+
+> A controllable device (like a lamp) would have a list of `in_services` that would indicate the services it supports. A controller (like a button) would have a list of `out_services` instead.
+
+## Definitions
+* `association_members` is an object describing association group members:
+ 
+| Field        | Type      | Example           | Description                                    |
+|--------------|-----------|-------------------|------------------------------------------------|
+| `group`      | int       | `1, 3, 5`         | Id of the supported group.                     |
```

Comment:
It should be `1`, `3`, `5`

Reply:
Also maybe no need to put multiple examples, just one is enough?

Reply:
One is enough, but I like to put at least 2 just to show generally what kind of numbers will be there so, for example, in case of group it's probably a low number and command class is something closer to a 100